controllers/storagecluster/cephcluster.go

```diff
@@ -454,9 +455,9 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, kmsConfigMap *co
 				AllowOsdCrushWeightUpdate:    true,
 				StorageClassDeviceSets:       newStorageClassDeviceSets(sc),
 				FlappingRestartIntervalHours: 24,
-				FullRatio:                    sc.Spec.ManagedResources.CephCluster.FullRatio,
-				NearFullRatio:                sc.Spec.ManagedResources.CephCluster.NearFullRatio,
-				BackfillFullRatio:            sc.Spec.ManagedResources.CephCluster.BackfillFullRatio,
+				NearFullRatio:                getFullRatios(sc, "nearFull"),
```

Comment:
Based on the last discussion in the [epic](https://issues.redhat.com/browse/RHSTOR-5929), we want to leave these values as null in the CephCluster CR for the default. So we would need the following behavior:
- If the ratios are not set in the StorageCluster CR, leave the default as null in the CephCluster CR
- The default values still need to be set on the prometheus rules, of course

This allows customers who previously had a support exception to change the values via the odf cli to continue using those values, instead of being required to set them in the StorageCluster CR as part of the 4.17 upgrade. But we would still encourage the customers to set them in the StorageCluster CR so they get the proper thresholds set in the alerts.

Reply:
As per the above comment, isn't it better to use `sc.Spec.ManagedResources.CephCluster.NearFull (and other ratios) directly? We can use the `getFullRatio()` function to set the PrometheusRule file values.

Reply:
@travisn missed the latest discussion about upgrade there. Have updated accordingly now.